contrib/pkg/createcluster/create.go

```diff
@@ -278,8 +290,9 @@ create-cluster CLUSTER_DEPLOYMENT_NAME --cloud=ovirt --ovirt-api-vip 192.168.1.2
 	flags.StringVar(&opt.HibernateAfter, "hibernate-after", "", "Automatically hibernate the cluster whenever it has been running for the given duration")
 	flags.StringVar(&opt.PullSecretFile, "pull-secret-file", defaultPullSecretFile, "Pull secret file for cluster")
 	flags.StringVar(&opt.BoundServiceAccountSigningKeyFile, "bound-service-account-signing-key-file", "", "Private service account signing key (often created with ccoutil create key-pair)")
-	flags.BoolVar(&opt.CredentialsModeManual, "credentials-mode-manual", false, "Configure the Cloud Credential Operator in the target cluster to Manual mode. Implies the use of --manifests to inject custom Secrets for all CredentialsRequests in the cluster.")
-
+	flags.BoolVar(&opt.CredentialsModeManual, "credentials-mode-manual", false, fmt.Sprintf(`Configure the Cloud Credential Operator in the target cluster to Manual mode.
+Implies the use of --manifests to inject custom Secrets for all CredentialsRequests in the cluster.
+Manual credentials mode is enabled by default and is required for the following clouds: %s`, strings.Join(manualModeClouds, "|")))
```

Comment:
The wording of this is a bit confusing. Defaulting implies that there's another option; "required" implies you have to specify the argument, which somewhat contradicts the defaulting and also I think isn't true. What about:
```suggestion
This option is redundant (but permitted) for following clouds, which always use manual mode: %s`, strings.Join(manualModeClouds, "|")))
```